packages/editor-ui/src/stores/posthog.ts

```diff
@@ -157,6 +161,10 @@ export const usePostHog = defineStore('posthog', () => {
 		if (name === ONBOARDING_EXPERIMENT.name && variant === ONBOARDING_EXPERIMENT.variant) {
 			segmentStore.showAppCuesChecklist();
 		}
+
+		if (name === TEMPLATE_EXPERIMENT.name && variant === TEMPLATE_EXPERIMENT.variant) {
```

Comment:
does not this need to be imported? in testing, does not seem this feature works

Reply:
Not sure how that happened. I might have forgotten to commit something because I tested it locally, and it worked fine. I just added the import